Framework/PythonInterface/plugins/functions/Bessel.py

```diff
@@ -0,0 +1,35 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+# pylint: disable=invalid-name, anomalous-backslash-in-string, attribute-defined-outside-init
+
+"""
```

Comment:
Can this be removed, this information is available via Git.

Reply:
I suppose you referred to lines 6 and 7?